cloud/google/cmd/gce-machine-controller/main.go

```diff
@@ -68,3 +74,10 @@ func main() {
 	c.RunAsync(shutdown)
 	select {}
 }
+
+func newConfigWatchOrNil() (*machinesetup.ConfigWatch, error) {
```

Comment:
This main.go is for the pod machine controller. It should always have a non-empty path passed in. 

Empty path is only for deployer tool use of machine actuator.

Reply:
Got it, i will remove the check for the empty path since that is an invalid state and we would prefer an error.